22_pipeslab/pipeslab.txt

```diff
@@ -2,20 +2,44 @@ For each of the following, type a single command line that performs:
 
   1. Print the number of lines from "ls -l" which represent a path that has execute permission to their owner.
   Answer: 
+	ls -l|grep $USER| cut -c 4|grep x|wc -w
+	[Q: Based on the commands we know by now is there a shorter way to do 	this? With less pipes?] 
 
   2. Print the number of lines from "ls -l" which represent a path that has execute permission to their owner, group and others.
   Answer: 
+	ls -l|grep $USER| cut -c 4,7,10|grep x|wc -w
 
   3. Print the number of lines from "ls -l" which represent paths that were modified today (no matter which day we run the command)
   Answer: 
+	
+	ls -l | cut -f6-8 -d" "| grep $(date +%b" "%d)| wc -l
```

Comment:
1. In this case I think we had no extra space in the output, That's the reason I didn't use tr
2. Thanks for this tip